### DISCUSSION TITLE
different array types

### DISCUSSION THREAD

**Discussion by rajamoulimallareddy** [prisma/prisma#17466] [Q&A]
is there any other way to put like `any[]` instead of `string[]`
i want like array in array.
how to do it?

**Comment by nurul3101** [comment]
Hey @rajamoulimallareddy 👋 

Can you please elaborate and provide an example of what you are trying to achieve?
Which database are you using?
Is your issue related to the types that are generated by Prisma?
Maybe, sharing the schema file could also be helpful.

**Reply by rajamoulimallareddy** [reply]
I want to save the data of array Inside array instead of string inside array.

**Reply by nurul3101** [reply]
Multi-Dimensional Arrays are not supported natively in Prisma.
- https://github.com/prisma/prisma/issues/11120

You would need to use [JSON](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#json) field for now

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
